pkg/controller/nodeipam/ipam/cidrset/metrics.go

```diff
@@ -44,6 +44,15 @@ var (
 		},
 		[]string{"clusterCIDR"},
 	)
+	cidrSetMax = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      nodeIpamSubsystem,
+			Name:           "cidrset_cidrs_max_total",
```

Comment:
`cidrset_cidrs_limit` should be easier to understand, but if you want to stick to a maximum to keep it coherent with the API, then I would suggest `cirdset_max_cidrs`